config/locales/en.yml

```diff
@@ -76,7 +76,7 @@ en:
       edit_category: 'Edit category'
     create:
       category_created: 'You successfully create category.'
-      category_not_created: 'Category not create'
+      category_not_created: 'Category cannot be created. Please enter more than 2 characters.'
```

Comment:
@sinpai I think it's bad errors message. if I enter more than 40 characters I get an error "...enter more than 2 characters". Let's change it.